maintainers/maintainer-list.nix

```diff
@@ -11707,6 +11707,12 @@
     github = "mihnea-s";
     githubId = 43088426;
     name = "Mihnea Stoian";
+  };
+    mikecm = {
```

Comment:
This line is indented 2 spaces too much.

Reply:
And the whole entry must be moved down to be properly sorted (as pointed out by the CI failure).

Reply:
Done and done.

Reply:
And squash commits.

Reply:
How do I do that?
A `git merge --squash` in my clone (on the appropriate branch)?
then push..

Reply:
I usually do `git rebase -i master` and change "pick" lines to "squash" lines.